app/upgrades/signetlaunch/upgrades.go

```diff
@@ -23,10 +23,17 @@ import (
 	btclighttypes "github.com/babylonlabs-io/babylon/x/btclightclient/types"
 )
 
+const (
+	ZoneConciergeStoreKey = "zoneconcierge"
+)
+
 var Upgrade = upgrades.Upgrade{
 	UpgradeName:          "signet-launch",
 	CreateUpgradeHandler: CreateUpgradeHandler,
-	StoreUpgrades:        store.StoreUpgrades{},
+	// Upgrade necessary for deletions of `zoneconcierge`
+	StoreUpgrades: store.StoreUpgrades{
+		Deleted: []string{ZoneConciergeStoreKey},
```

Comment:
Have we ever tested if adding this store back to another upgrade with the same key works? 
Asking this, because I never saw to delete a module to add it back later and not sure of the implications that might have 

Also, the worst case would be when adding it back with a different naming than `zoneconcierge`

Reply:
tbh  later I would re-add it with different key, even `zoneconcierge1` would be fine just to avoid thinking about this case 😅